nixos/modules/services/networking/3proxy.nix

```diff
@@ -18,26 +18,26 @@ in {
       type = types.nullOr types.path;
       default = null;
       example = "/var/lib/3proxy/3proxy.passwd";
-      description = ''
+      description = lib.mdDoc ''
         Load users and passwords from this file.
 
         Example users file with plain-text passwords:
 
-        <programlisting>
+        ```
           test1:CL:password1
           test2:CL:password2
-        </programlisting>
+        ```
 
         Example users file with md5-crypted passwords:
 
-        <programlisting>
+        ```
           test1:CR:$1$tFkisVd2$1GA8JXkRmTXdLDytM/i3a1
           test2:CR:$1$rkpibm5J$Aq1.9VtYAn0JrqZ8M.1ME.
-        </programlisting>
+        ```
 
         You can generate md5-crypted passwords via https://unix4lyfe.org/crypt/
         Note that htpasswd tool generates incompatible md5-crypted passwords.
-        Consult <link xlink:href="https://github.com/z3APA3A/3proxy/wiki/How-To-(incomplete)#USERS">documentation</link> for more information.
+        Consult [documentation](https://github.com/z3APA3A/3proxy/wiki/How-To-(incomplete)#USERS) for more information.
```

Comment:
This might be problematic in some markdown processors including the official commonmark one: https://babelmark.github.io/?text=%5Bdocumentation%5D(https%3A%2F%2Fgithub.com%2Fz3APA3A%2F3proxy%2Fwiki%2FHow-To-(incomplete)%23USERS

The parens should be urlencoded `https://github.com/3proxy/3proxy/wiki/How-To-%28incomplete%29#USERS`